packages/frontend/src/utils/networks.ts

```diff
@@ -69,3 +71,26 @@ export function isL2ToL1(srcNetwork: Network, destNetwork: Network) {
 
   return false
 }
+
+export function isProviderNetworkByChainSlugOrId(
```

Comment:
suggestion:

Remove the slug type and just use chainId for both of this methods